src/js/components/stock-movement-wizard/outbound/AddItemsPage.jsx

```diff
@@ -686,24 +686,6 @@ class AddItemsPage extends Component {
     });
   }
 
-  confirmSubmit(onConfirm) {
-    confirmAlert({
-      title: this.props.translate('react.stockMovement.message.confirmSubmit.label', 'Confirm submit'),
-      message: this.props.translate(
-        'react.stockMovement.confirmSubmit.message',
-        'Please confirm you are ready to submit your request. Once submitted, you cannot edit the request.',
-      ),
-      buttons: [
-        {
-          label: this.props.translate('react.default.goBack.label', 'Go back'),
-        },
-        {
-          label: this.props.translate('react.default.submit.label', 'Submit'),
-          onClick: onConfirm,
-        },
-      ],
-    });
-  }
```

Comment:
I saw the same label being used in the outbound workflow, so I wanted to change it here, but it turned out that this `confirmSubmit` function is not even used in this component (probably a copy paste from the request was done here)